docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff permaddr 14:02:ec:7f:3a:3c
+4: enp2s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
+    link/ether 0c:20:12:fe:01:00 brd ff:ff:ff:ff:ff:ff
+10: bond0: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+13: bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+```
+
+Confirm that the bonding driver is reporting the selected primary interface:
+```console
+cat /proc/net/bonding/bond0
+
+Ethernet Channel Bonding Driver: v6.6.74-flatcar
+
+Bonding Mode: fault-tolerance (active-backup)
+Primary Slave: enp2s1 (primary_reselect always)
+Currently Active Slave: enp2s1
+MII Status: up
+MII Polling Interval (ms): 50
+Up Delay (ms): 0
+Down Delay (ms): 0
+Peer Notification Delay (ms): 0
+
+Slave Interface: enp2s1
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: a0:36:9f:3f:1a:e8
+Slave queue ID: 0
+
+Slave Interface: enp2s2
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 14:02:ec:7f:3a:3c
+Slave queue ID: 0
+```
+
+### Failure Scenarios
+When `miimon` detects that the primary slave is no longer `up`, the bonding
+driver will send 5 gratitious arps via the secondary slave to notify the switch
+that the IP address is now on a new switch. The fabric will start routing
+traffic to this location and the host will start receiving traffic.
+
+If the `miimon` polling determines that the primary slave link is `up` the
+bonding driver will switch back to the primary slave. This behavior is
+configurable, see the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation for additional details.
+
+The change over process will result in dropped packets, but should not result
+in TCP timeouts. 
+
+## VLANS
+
+A Linux host is capable of receiving tagged and untagged traffic. Saying the
+same thing, another way, a Linux host can be attached to a trunk or access port
+on a switch.
+
+The Open Network Fabric is capable of emitting tagged and untagged traffic. The
+setting to change if traffic is tagged or untagged is on the
+[VPCAttahcment](vpcs.md#vpcattachment), specifically the `nativeVLAN` field.
+The default value of this field is `false`, meaning the fabric will emit tagged
+traffic. This means that the Linux host must handle the tag, this is
+accomplished by creating a link layer interface on the host for the tag:
+```bash
+sudo ip link add link enp2s1 name enp2s1.1001 type vlan id 1001
+```
+The name `enp2s1.1001` is by convention, it can be customized. The `vlan` in
+this example is `1001` that is created when the subnet is created inside the
```

Comment:
```suggestion
The name `enp2s1.1001` is by convention, it can be customized. The `vlan id` in
this example is `1001` and was created when the subnet was created inside the
```